erts/test/lux/input/feature_directive_maybe_ifn.erl

```diff
@@ -0,0 +1,87 @@
+%% This module knows about the both features ifn and maybe
+%% These atoms are thus quoted
+
+-module(feature_directive_maybe_ifn).
+
+-export([foo/0,
+	 bar/0,
```

Comment:
for some reason your exports contain tab characters (other files as well), while spaces are used in other places ...

Reply:
Strange.  I'll take a look at this and correct it.